lib/actions/model.js

```diff
@@ -61,4 +61,26 @@ module.exports = {
       return xhr;
     };
   },
+
+  validate(brainstemKey, modelId, attributes, options = {}) {
+    const {
+      validateOptions = {},
+      preValidateAction,
+      postValidateAction,
+      trackKey,
```

Comment:
`trackKey` is not being used? 

It is usually used to cancel previous XHRs but validation is synchronous.